bin/mycroft-stop

```diff
@@ -14,8 +14,107 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SOURCE="${BASH_SOURCE[0]}"
-cd -P "$( dirname "$SOURCE" )"/.. || exit
-DIR="$( pwd )"
+script=${0}
+script=${script##*/}
 
-. "$DIR/stop-mycroft.sh" "$@"
+help() {
+	echo "${script}:  Mycroft service stopper"
+	echo "usage: ${script} [service]"
+	echo
+	echo "Service:"
+	echo "  all       ends core services: bus, audio, skills, voice"
+	echo "  (none)    same as \"all\""
+	echo "  bus       stop the Mycroft messagebus service"
+	echo "  audio     stop the audio playback service"
+	echo "  skills    stop the skill service"
+	echo "  voice     stop voice capture service"
+	echo "  enclosure stop enclosure (hardware/gui interface) service"
+	echo
+	echo "Examples:"
+	echo "  ${script}"
+	echo "  ${script} audio"
+
+	exit 0
+}
+
+process_running() {
+	if [ "$( pgrep -f "python3 (.*)-m mycroft.*${1}" )" ]; then
+		return 0
+	else
+		return 1
+	fi
+}
+
+end_process() {
+	if process_running "$1"; then
+		# Find the process by name, only returning the oldest if it has children
+		pid=$( pgrep -o -f "python3 (.*)-m mycroft.*${1}" )
+		echo "Stopping $1 (${pid})..."
+		kill -SIGINT "${pid}"
```

Comment:
For me this doesn't work:
```
53: kill: Illegal option -S
```
Should it be `kill -s SIGINT` ?

Reply:
I believe I changed it from that because that didn't work for someone else... I have no problem changing it back though, both options work for me. Changed.